### DISCUSSION TITLE
Portal for all interview experiences with question approaches, editorials, practice link(company specific)

### DISCUSSION THREAD

**Discussion by rishabhrishabh** [scaleracademy/open-source-projects#56] [Ideas]
**IDEA OBJECTIVE**
Basic idea is to create a portal on Interviewbit which consists of all the interview experiences company-wise with their editorials and Interviewbit practice link. User can also add their interview experience on the portal with some format which will be defined by Interviewbit. All the interview experience of Company A will be under the tag of company A, suppose the user wants to see the interview experience of company A, he can open that, there he will all the questions with approaches(editorial) also. This is a new thing, where we are adding approaches, as most of the students don't know about solutions and they will leave that question if they cant come upon a solution after trying many times.

**USE CASES**
- Suppose a user wants to see the interview experiences of the company then he can go to the portal and can check the experience with the approaches of all questions and their editorial and practice link. In this way if new questions came in any interview experience then it can be created by problem setter on Interviewbit and question in question bank of Interviewbit also increases.
- Also if anyone wants to add their interview experience and wants to help students they can do so by adding their experience on the portal in a particular format decided by Scaler.

**Comment by Vasu201499** [comment]
Good one

**Comment by Vikas19556** [comment]
Nice

**Comment by Anmol-08** [comment]
Nice idea

**Comment by dipali0269** [comment]
Great initiative!!

**Comment by championswimmer** [comment]
Doesn't CareerCup already do this ?

**Reply by rishabhrishabh** [reply]
Hi @championswimmer 
No, actually there any one can give their answers and they are not verified and correctness totally depend on upvotes and their questions have company tags.
What my idea is that scaler will collect interview experience from users in particular format round wise and then if the questions in interview experience is already on interviewbit then we will provide link of their , if not then interviewbit problem setters will set that problem and write it's approaches or pseudo code also and that will be provided with the interview experience. In this way question bank of IB will also scale up and users will get interview experiences of all companies with their approaches or pseudo codes. 
This is like portal where you will get all interview experiences of a company with approaches and codes. Company is parent and all the interview experiences of that particular company is child of that means Amazon will contain all interview experiences of Amazon. Here our approaches or solutions of questions are verified by IB.

**Reply by jeswanth9999** [reply]
Will this be a copy right issue? Just asking to know.

**Reply by rishabhrishabh** [reply]
No, I don't think so there will be any copyright issue as user is themselves giving their questions, moreover if there is any copyright issue to tell interview questions then on YouTube and on other portals no interview questions should be there.

**Comment by jeswanth9999** [comment]
Thanks for the information. Just ask out of curiosity that people choose Leetcode for interview questions why not other popular platform's as it. They can just maintain same data picking from Leetcode but don't do this. Again, out of curiosity it is. To give small info I don't use Leetcode so I dont' know how it actually works.
